Assets/Scripts/Player/PlayerShooting.cs

```diff
@@ -11,18 +11,37 @@ public sealed class PlayerShooting : MonoBehaviour
 
     private GameObject _bullet;
     private float _shootingPower;
+    private float _shootingDelay;
```

Comment:
```suggestion
    private float _maxShootingDelay;
```
Als ik kijk naar hoe deze variable gebruikt wordt in het script denk ik dat het logischer is dat dit het woord "max" ervoor heeft.